components/cam/chem_proc/src/cam_chempp/mozpp.main.f

```diff
@@ -173,7 +173,8 @@ program mozart_pp
       sim_dat_filename = 'sim.dat'
       sim_dat_filespec = trim(sim_dat_path) // 'sim.dat'
       cpp_dir  = ' '
-      cpp_opts = '-P -C -I.'
+!     cpp_opts = '-P -C -I.'  ! -C causes problems on Cori
+      cpp_opts = '-P -I. -traditional-cpp'  ! Works on Cori (2019-07-15)
```

Comment:
I think we can just remove the `-C` flag. Also, we should just get rid of the commented-out line.

Reply:
Did you diff the fortran files to those archived in src/chemistry ?

Reply:
My personal preference is to leave in (as comments) things that may be relevant to getting the pre-processor to work an another machine.  It is a cheap alternative to IF statements for different machines.

Reply:
Agreed. Lets keep it the way it is.